internal/driver/driver.go

```diff
@@ -134,7 +141,8 @@ func (d *Driver) HandleReadCommands(deviceName string, protocols map[string]mode
 	for i, req := range reqs {
 		res, err := handleReadCommandRequest(deviceClient, req)
 		if err != nil {
-			driver.Logger.Errorf("Read command failed. Cmd:%v err:%v \n", req.DeviceResourceName, err)
+			driver.Logger.Errorf("Read command failed, remove the Modbus client from the cache to allow re-establish connection next time. Cmd: %v err:%v", req.DeviceResourceName, err)
+			//d.removeDeviceClient(connectionInfo)
```

Comment:
why does it comment out?

Reply:
I forgot to uncomment is after testing. Updated.